src/class-parsely.php

```diff
@@ -362,9 +362,10 @@ public function get_options() {
 		 *
 		 * @var Parsely_Options|null
 		 */
-		$options = get_option( self::OPTIONS_KEY, $this->option_defaults );
+		$options = get_option( self::OPTIONS_KEY, null );
 
 		if ( ! is_array( $options ) ) {
+			$this->set_default_track_as_values();
```

Comment:
Just to confirm: This will only effect new plugin istalls and doesn't change anything for existing installs ... right?

Reply:
Yes. The idea here is to set the default values only if the options are unset or invalid. This won't affect existing installs, as the settings are already an array there.

Reply:
I've pushed a test that verifies that the `set_default_track_as_values()` function doesn't fire when options are saved.